knock/src/components/FeedCard/FeedCard.tsx

```diff
@@ -0,0 +1,77 @@
+import { QuestionDetailResponse } from '../../core/types/api/Response';
+import Badge from '../../core/ui/Badge/Badge';
+import Dropdown from '../../core/ui/Dropdown/Dropdown';
+import Question from '../Question/Question';
+import Reaction from '../../core/ui/Reaction/Reaction';
+import styles from './FeedCard.module.scss';
+
+interface FeedCardProps extends QuestionDetailResponse {
+  handleUpdateAnswer: (answerId: number | undefined, content: string) => void;
+  handleRejectAnswer: (
+    answerId: number | undefined,
+    isRejected: boolean,
+  ) => void;
+  handleClickLike: (questionId: number) => void;
+  handleClickDislike: (questionId: number) => void;
+}
+
+const FeedCard = ({
+  id,
+  subjectId,
+  handleUpdateAnswer,
+  handleRejectAnswer,
+  content,
+  like,
+  dislike,
+  handleClickLike,
+  handleClickDislike,
+  createdAt,
+  answer,
```

Comment:
rest 연산자를 통해서 보기 편하게 사용할 수 있습니다 😄
```suggestion
  handleUpdateAnswer,
  handleRejectAnswer,
  handleClickLike,
  handleClickDislike,
 ...questionDetails,
```

Reply:
감사합니다!